I 03-17 20:25:51 controller.py:59] DAG:
I 03-17 20:25:51 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 20:25:51 controller.py:59]   resources: AWS(cpus=4, {'L4': 1})]
I 03-17 20:25:51 controller.py:195] Submitted managed job 305 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-20-25-51-852979_compute-text-vectors_305-0
I 03-17 20:25:51 controller.py:199] Started monitoring.
I 03-17 20:25:51 state.py:451] Launching the spot cluster...
I 03-17 20:25:52 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 20:25:53 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:25:53 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:25:53 optimizer.py:749] 
I 03-17 20:25:53 optimizer.py:884] Considered resources (1 node):
I 03-17 20:25:53 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:25:53 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:25:53 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:25:53 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-1     0.80       [32m   âœ”[0m     
I 03-17 20:25:53 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:25:53 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a,us-east-1b,us-east-1c,us-east-1d).
[2mW 03-17 20:25:55 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:02 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:03 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:26:03 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:26:03 cloud_vm_ray_backend.py:2097] 
W 03-17 20:26:03 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:26:03 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:26:03 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:26:03 optimizer.py:749] 
I 03-17 20:26:03 optimizer.py:884] Considered resources (1 node):
I 03-17 20:26:03 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:26:03 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:26:03 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:26:03 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-2     0.80       [32m   âœ”[0m     
I 03-17 20:26:03 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:26:04 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a,us-east-2b,us-east-2c).
[2mW 03-17 20:26:07 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:08 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:09 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:26:09 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:26:09 cloud_vm_ray_backend.py:2097] 
W 03-17 20:26:09 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:26:09 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:26:09 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:26:09 optimizer.py:749] 
I 03-17 20:26:09 optimizer.py:884] Considered resources (1 node):
I 03-17 20:26:09 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:26:09 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:26:09 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:26:09 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-west-2     0.80       [32m   âœ”[0m     
I 03-17 20:26:09 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:26:10 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2a,us-west-2b,us-west-2c,us-west-2d).
[2mW 03-17 20:26:14 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:21 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:33 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:29:48 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:29:48 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-west-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:29:48 cloud_vm_ray_backend.py:2097] 
W 03-17 20:29:48 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:29:48 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:29:48 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 20:29:48 optimizer.py:749] 
I 03-17 20:29:48 optimizer.py:884] Considered resources (1 node):
I 03-17 20:29:48 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:29:48 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:29:48 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:29:48 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-north-1    0.85       [32m   âœ”[0m     
I 03-17 20:29:48 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:29:48 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a,eu-north-1b).
[2mW 03-17 20:29:55 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:29:58 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:29:59 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-north-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:29:59 cloud_vm_ray_backend.py:2097] 
W 03-17 20:29:59 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:29:59 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:29:59 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 20:29:59 optimizer.py:749] 
I 03-17 20:29:59 optimizer.py:884] Considered resources (1 node):
I 03-17 20:29:59 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:29:59 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 20:29:59 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:29:59 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ca-central-1   0.89       [32m   âœ”[0m     
I 03-17 20:29:59 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:29:59 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a,ca-central-1b).
[2mW 03-17 20:30:02 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:03 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:30:03 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ca-central-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:03 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:03 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:04 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:04 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:30:04 optimizer.py:749] 
I 03-17 20:30:04 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:04 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:04 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:30:04 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:04 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-south-1    0.97       [32m   âœ”[0m     
I 03-17 20:30:04 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:04 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a,ap-south-1b).
[2mW 03-17 20:30:13 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1b.[0m
[2mW 03-17 20:30:16 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1a.[0m
W 03-17 20:30:17 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-south-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:17 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:17 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:17 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:17 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:30:17 optimizer.py:749] 
I 03-17 20:30:17 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:17 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:30:17 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 20:30:17 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:30:17 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-2   0.99       [32m   âœ”[0m     
I 03-17 20:30:17 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:17 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a,ap-northeast-2d).
[2mW 03-17 20:30:26 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2d.[0m
[2mW 03-17 20:30:28 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2d). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2a.[0m
W 03-17 20:30:29 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-northeast-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:29 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:29 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:29 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:29 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:30:29 optimizer.py:749] 
I 03-17 20:30:29 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:29 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:30:29 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 20:30:29 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:30:29 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-central-1   1.01       [32m   âœ”[0m     
I 03-17 20:30:29 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:30 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1a,eu-central-1b).
[2mW 03-17 20:30:36 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:38 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:30:39 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-central-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:39 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:39 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:39 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:39 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:30:39 optimizer.py:749] 
I 03-17 20:30:39 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:39 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:39 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:30:39 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:39 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-2     1.02       [32m   âœ”[0m     
I 03-17 20:30:39 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:39 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a,eu-west-2b).
[2mW 03-17 20:30:44 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:45 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:30:46 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:46 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:46 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:46 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:46 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:30:46 optimizer.py:749] 
I 03-17 20:30:46 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:46 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:46 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:30:46 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:46 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-3     1.02       [32m   âœ”[0m     
I 03-17 20:30:46 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:46 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b,eu-west-3c).
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:31:15 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:31:42 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-305.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 20:31:42 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:31:42 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-apznrqrs/job-apznrqrs/workdir -> ~/sky_workdir[0m
I 03-17 20:31:45 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:31:45 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final -> /output[0m
I 03-17 20:31:48 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-25-52-486622/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 20:31:50 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 20:31:52 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 20:31:52 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 20:31:56 utils.py:131] === Checking the job status... ===
I 03-17 20:31:56 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:31:56 utils.py:140] ==================================
I 03-17 20:31:57 state.py:471] Job started.
I 03-17 20:32:17 utils.py:131] === Checking the job status... ===
I 03-17 20:32:18 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:32:18 utils.py:140] ==================================
I 03-17 20:32:38 utils.py:131] === Checking the job status... ===
I 03-17 20:32:39 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:32:39 utils.py:140] ==================================
I 03-17 20:32:59 utils.py:131] === Checking the job status... ===
I 03-17 20:33:00 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:33:00 utils.py:140] ==================================
I 03-17 20:33:20 utils.py:131] === Checking the job status... ===
I 03-17 20:33:21 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:33:21 utils.py:140] ==================================
I 03-17 20:33:41 utils.py:131] === Checking the job status... ===
I 03-17 20:33:42 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:33:42 utils.py:140] ==================================
I 03-17 20:34:02 utils.py:131] === Checking the job status... ===
I 03-17 20:34:03 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:34:03 utils.py:140] ==================================
I 03-17 20:34:23 utils.py:131] === Checking the job status... ===
I 03-17 20:34:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:34:24 utils.py:140] ==================================
I 03-17 20:34:44 utils.py:131] === Checking the job status... ===
I 03-17 20:34:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:34:45 utils.py:140] ==================================
I 03-17 20:35:05 utils.py:131] === Checking the job status... ===
I 03-17 20:35:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:35:06 utils.py:140] ==================================
I 03-17 20:35:26 utils.py:131] === Checking the job status... ===
I 03-17 20:35:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:35:27 utils.py:140] ==================================
I 03-17 20:35:47 utils.py:131] === Checking the job status... ===
I 03-17 20:35:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:35:48 utils.py:140] ==================================
I 03-17 20:36:08 utils.py:131] === Checking the job status... ===
I 03-17 20:36:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:36:09 utils.py:140] ==================================
I 03-17 20:36:29 utils.py:131] === Checking the job status... ===
I 03-17 20:36:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:36:30 utils.py:140] ==================================
I 03-17 20:36:50 utils.py:131] === Checking the job status... ===
I 03-17 20:36:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:36:51 utils.py:140] ==================================
I 03-17 20:37:11 utils.py:131] === Checking the job status... ===
I 03-17 20:37:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:37:12 utils.py:140] ==================================
I 03-17 20:37:32 utils.py:131] === Checking the job status... ===
I 03-17 20:37:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:37:33 utils.py:140] ==================================
I 03-17 20:37:53 utils.py:131] === Checking the job status... ===
I 03-17 20:37:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:37:56 utils.py:140] ==================================
I 03-17 20:38:17 utils.py:131] === Checking the job status... ===
I 03-17 20:38:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:38:19 utils.py:140] ==================================
I 03-17 20:38:40 utils.py:131] === Checking the job status... ===
I 03-17 20:38:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:38:41 utils.py:140] ==================================
I 03-17 20:39:02 utils.py:131] === Checking the job status... ===
I 03-17 20:39:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:02 utils.py:140] ==================================
I 03-17 20:39:22 utils.py:131] === Checking the job status... ===
I 03-17 20:39:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:23 utils.py:140] ==================================
I 03-17 20:39:43 utils.py:131] === Checking the job status... ===
I 03-17 20:39:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:44 utils.py:140] ==================================
I 03-17 20:40:04 utils.py:131] === Checking the job status... ===
I 03-17 20:40:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:40:05 utils.py:140] ==================================
I 03-17 20:40:25 utils.py:131] === Checking the job status... ===
I 03-17 20:40:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:40:26 utils.py:140] ==================================
I 03-17 20:40:46 utils.py:131] === Checking the job status... ===
I 03-17 20:40:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:40:47 utils.py:140] ==================================
I 03-17 20:41:07 utils.py:131] === Checking the job status... ===
I 03-17 20:41:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:41:08 utils.py:140] ==================================
I 03-17 20:41:28 utils.py:131] === Checking the job status... ===
I 03-17 20:41:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:41:29 utils.py:140] ==================================
I 03-17 20:41:49 utils.py:131] === Checking the job status... ===
I 03-17 20:41:49 utils.py:137] Job status: JobStatus.SUCCEEDED
I 03-17 20:41:49 utils.py:140] ==================================
I 03-17 20:41:50 state.py:561] Job succeeded.
I 03-17 20:41:50 controller.py:238] Managed job 305 (task: 0) SUCCEEDED. Cleaning up the cluster compute-text-vectors-305.
E 03-17 20:41:52 controller_utils.py:376] Failed to find the logs for the user program at ~/sky_logs/managed_jobs/sky-2025-03-17-20-25-52-486622/run.log.
I 03-17 20:41:52 controller.py:117] 
I 03-17 20:41:52 controller.py:117] == End of logs (ID: 305) ==
I 03-17 20:42:00 state.py:656] Cancellation skipped, job is already terminal
I 03-17 20:42:00 state.py:677] Cancellation skipped, job is not CANCELLING
I 03-17 20:42:02 controller.py:552] Killing controller process 161088.
I 03-17 20:42:02 controller.py:560] Controller process 161088 killed.
I 03-17 20:42:02 controller.py:562] Cleaning up any cluster for job 305.
I 03-17 20:42:02 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-apznrqrs
<sky-payload>"<rich_init>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-apznrqrs[/][/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-apznrqrs[/][/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-apznrqrs[/][/]</rich_start>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 20:42:03 storage.py:1505] [32mDeleted S3 bucket skypilot-filemounts-root-726f1974-apznrqrs.[0m
I 03-17 20:42:03 controller.py:571] Cluster of managed job 305 has been cleaned up.
